solutions.js

```diff
@@ -3,3 +3,14 @@ console.log('Defining functions!');
 function changeNavColor() {
   // change the navbar to a different color
 };
+
+document.getElementById("topbar").style.background = 'red';
+
+document.querySelector("body").innerHTML = "<h1>This Page is now under Devin's control!<h1>"+document.querySelector("body").innerHTML
```

Comment:
A more specific way to grab that top bar is `document.querySelector('#topbar');`